templates/templates/kyma-integration.yaml

```diff
@@ -370,6 +370,9 @@ postsubmits:
         testgrid-create-test-group: "false"
       branches:
         - ^master$
+      {{if eq .channel "rapid"}}
```

Comment:
{{- if eq .channel "rapid"}}
Try add dash on front to get rid empty line.

Reply:
Fixed, thanks! :)